anvil/src/config.rs

```diff
@@ -715,7 +715,7 @@ impl NodeConfig {
                 env.block = BlockEnv {
                     number: fork_block_number.into(),
                     timestamp: block.timestamp,
-                    difficulty: block.difficulty,
+                    difficulty: Default::default(),
```

Comment:
should this remain as `block.difficulty`?

Reply:
oh right, this is not a good change since this affects the `block.difficulty` var, 

my thinking here was if we remove it here, it will essentially freeze the `totalDifficulty`.

what should we do with the difficulty after the first block "mined" by anvil itself, should we just continue to use the `difficulty` of the block we forked off or `0`.

if we continue to use the difficulty of the forked block, or something else non zero then this PR needs a bit more work because then we need to track the totalDifficulty on a per block basis.

Reply:
i see. let's give the 'simpler' approach a try and see if anybody complains..